lazy.go

```diff
@@ -51,7 +51,11 @@ type lazyConn struct {
 }
 
 func (l *lazyConn) Read(b []byte) (int, error) {
-	if !l.rhandshake {
+	l.rhlock.RLock()
+	rhandshake := l.rhandshake
+	l.rhlock.RUnlock()
```

Comment:
We can't use `internal/race` internal packages are locked and can be used only used buy children of the package. In this case only buy stdlib.